cartography/intel/gcp/crm.py

```diff
@@ -45,14 +49,119 @@ def load_gcp_organizations(neo4j_session, data, gcp_update_tag):
         )
 
 
+def load_gcp_folders(neo4j_session, data, gcp_update_tag):
+    """
+    {'createTime': '2019-05-14T20:08:46.125Z',
+    'displayName': '<my folder name>',
+    'lifecycleState': 'ACTIVE',
+    'name': 'folders/<Folder ID>',
+    'parent': 'organizations/<Org ID>'},
+    """
+    for folder in data:
+        # Get the correct parent type.
+        # Parents of folders can only be GCPOrganizations or other folders, see
+        # https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy
+        if folder['parent'].startswith("organizations"):
+            query = "MATCH (parent:GCPOrganization{id:{ParentId}})"
+        elif folder['parent'].startswith("folders"):
+            query = """
+            MERGE (parent:GCPFolder{id:{ParentId}})
+            ON CREATE SET parent.firstseen = timestamp()
+            """
+        query += """
+        MERGE (folder:GCPFolder{id:{FolderName}})
+        ON CREATE SET folder.firstseen = timestamp()
+        SET folder.displayname = {DisplayName},
+        folder.lifecyclestate = {LifecycleState},
+        folder.lastupdated = {gcp_update_tag}
+        WITH parent, folder
+        MERGE (folder)-[r:PARENT]->(parent)
```

Comment:
should we do child->parent (reverse) or parent -> child (forward)? Most time people use forward for hierarchy